README.md

```diff
@@ -192,11 +192,23 @@ Install all the dependencies:
 
     $ yarn install
 
+#### The LTI frontend
 Run the build and copy the `iframe-resizer` host-side JS into your local `Django` assets:
 
-    $ yarn build
+    $ yarn build-lti
     $ yarn copy-iframe-resizer
 
+#### The "Standalone" frontend
+Run the build and copy the `iframe-resizer` host-side JS into your local `Django` assets:
```

Comment:
You don't need iframe-resizer for the standalone site (since the purpose is to have a site not embended in an iframe)

Reply:
I don't know how it got there...